src/Gemstone.IO/Collections/FileBackedHashSet.cs

```diff
@@ -220,7 +259,7 @@ public string FilePath
     /// Gets the default signature used by the <see cref="FileBackedHashSet{T}"/>
     /// if no user-defined signature is supplied.
     /// </summary>
-    public byte[] DefaultSignature => new Guid(FileBackedLookupTable<T, object>.HashSetSignature).ToRfcBytes();
+    public byte[] DefaultSignature => new Guid(FileBackedLookupTable<T, object, object, object>.HashSetSignature).ToRfcBytes();
```

Comment:
This doesn't really matter but seems more correct.
```suggestion
    public byte[] DefaultSignature => new Guid(FileBackedLookupTable<T, object, TElem, object>.HashSetSignature).ToRfcBytes();
```